src/inception.jl

```diff
@@ -0,0 +1,150 @@
+function conv_block(kernelsize::Tuple{Int64,Int64}, inplanes::Int64, outplanes::Int64; stride::Int64=1, pad::Union{Int64,Tuple{Int64,Int64}}=0)
+    conv_layer = []
+    push!(conv_layer, Conv(kernelsize, inplanes => outplanes, stride=stride, pad=pad))
+    push!(conv_layer, BatchNorm(outplanes, relu))
+    return conv_layer
+end
```

Comment:
It seems like this is a more general version of `conv_block` in "googlenet.jl". Can we add a "utilities.jl" file to collect utility functions like these that can be used across models? I think also renaming the function to `conv_bn` will be clearer.

Also, instead of creating an array, maybe we can just return a tuple?